torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -263,11 +263,13 @@ def get_edge_label_index(
 
     if isinstance(edge_label_index[0], str):
         edge_type = edge_label_index
+        edge_type = data._to_canonical(*edge_type)
         return edge_type, data[edge_type].edge_index
 
     assert len(edge_label_index) == 2
 
     edge_type, edge_label_index = edge_label_index
+    edge_type = data._to_canonical(*edge_type)
```

Comment:
not very happy accessing a "private" function - perhaps there is a better alternative

Reply:
Currently no. I agree with your concerns, but it is not blocking. Merging nonetheless.